django/db/models/base.py

```diff
@@ -387,7 +387,13 @@ class ModelState:
     # on the actual save.
     adding = True
     fields_cache = ModelStateFieldsCacheDescriptor()
+    prefetch_related_peers = ()
 
+    def __getstate__(self):
+        """Hook to allow choosing the attributes to pickle."""
+        res = dict(self.__dict__)
+        res['prefetch_related_peers'] = ()
```

Comment:
you can just `del` it instead, it will then fall back to the class level attribute, and save space on pickled state objects